src/web/tools/Vector.ts

```diff
@@ -3,28 +3,32 @@ import { MINIMAL_FLING_VELOCITY } from '../constants';
 import PointerTracker from './PointerTracker';
 
 export default class Vector {
-  x: number = 0;
-  y: number = 0;
-  unitX: number = 0;
-  unitY: number = 0;
+  x = 0;
+  y = 0;
+  unitX = 0;
+  unitY = 0;
+  magnitude = 0;
```

Comment:
```suggestion
  private readonly x;
  private readonly y;
  private readonly unitX;
  private readonly unitY;
  public readonly magnitude;
```

Since all of those are assigned only in constructor You can make them `readonly` . Also all of them (except `magnitude`) could be private, which makes me think about adding a getter.

Earlier you used `get magnitude` syntax which was fine (however now we would have to add something like `_magnitude` since it is used in constructor):

```jsx
  get magnitude(){
    return this._magnitude;
  }
```
Or we could simply add java style getter:

```jsx
  getMagnitude(){
    return this.magnitude;
  }
```

Reply:
What do we get from a getter here? It would be `readonly` anyway.

Reply:
I believe this is more matter of preference, for me it just looks better. That's why I asked about your opinion 😄